utils/create_validate_pr_quickstarts.ts

```diff
@@ -73,10 +89,17 @@ export const createValidateQuickstarts = async (
     )
     .flatMap((filePath) => {
       if (QUICKSTART_CONFIG_REGEXP.test(filePath)) {
-        return new Quickstart(filePath);
+        return new Quickstart(
+          filePath,
+          undefined,
```

Comment:
A comment here may be worth putting - we're basically "ignoring" the `basePath` argument ehre